backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/views/RV_DATEV_Export_Fact_Acct_Invoice.sql

```diff
@@ -0,0 +1,47 @@
+CREATE VIEW RV_DATEV_Export_Fact_Acct_Invoice (DebitOrCreditIndicator, Currency, dr_account, cr_account, amt, activityname, c_activity_id, documentno, dateacct, bpvalue, bpname, duedate, description, c_bpartner_id, c_invoice_id, docbasetype, fact_acct_id, rv_datev_export_fact_acct_invoice_id, ad_client_id, ad_org_id) AS
+SELECT CASE
```

Comment:
please update the existing ddl isntead of creating a new one: 
![image](https://user-images.githubusercontent.com/1264789/159854083-9e438a28-ba3a-4c2f-bcff-4db5a8d5603f.png)